Slight MySQL and MongoDB documentation changes

hhorak opened the pull request:
Added MongoDB general description and use THIS_IMAGE instead of {{ container }} value, since this name may change in the registry, during build or on user's machine.

Added MongoDB readme to be available in the container-image.

Merge two duplicate readme files in the MySQL/MariaDB readme files.

hhorak committed:
Slight MySQL and MongoDB documentation changes

Added MongoDB general description and use THIS_IMAGE instead of {{ container }} value, since this name may change in the registry, during build or on user's machine.

Added MongoDB readme to be available in the container-image.

Merge two duplicate readme files in the MySQL/MariaDB readme files.

hhorak commented:
@omron93 @mfojtik These should be only slight changes, cleaning and actually including mongodb specific help into container-usage output. But rather wait for your response if it's fine to be merged.

mfojtik commented on mysql-common/usr/share/cont-docs/30-mysql.txt.tpl:
````diff
@@ -32,36 +32,52 @@ You can also set following mount points by passing `-v /host:/container` flag to
+initialized, define them as environment variables
+
+```
+docker run -d --name mysql_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db THIS_IMAGE
````
Can you remove the --name here? If you start this container and then you stop it, I think the default Docker behavior is that it will refuse to start a new one until you remove the 'mysql_database' one ;-)

mfojtik replied:
You can pass `--cidfile=` option which will instruct Docker to write a file with the container id

mfojtik commented:
@hhorak LGTM (one nit about the name)

hhorak committed:
Remove --name options from the examples

If we start a container with --name mysql_database and then we stop it, the default Docker behavior is that it will refuse to start a new one until we remove the 'mysql_database'.

It is possible to get pass --cidfile= option which will instruct Docker to write a file with the container id, which may be handy for customers, so let's document it.

hhorak commented:
@mfojtik thx for the --name hint, I've removed it from the docs entirely and also added a comment about --cidfile to general usage message, it may be handy for users.
